app/controllers/admin/categories_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Admin::CategoriesController < ApplicationController
+  def new; end
+
+  def create
+    @category = Category.new category_params
+    if @category.save
+      flash[:success] = 'Create category success'
+      redirect_to root_path
+    else
+      flash[:success] = 'Create category failed'
```

Comment:
trường hợp không tạo được sao lại dùng `flash[:success]` và em test xem khi tạo lỗi thì có in ra flash không? Nếu không thì đọc rails flash rồi update lại

Reply:
oki a